core/modules/layout/layout.theme.inc

```diff
@@ -137,6 +137,19 @@ function theme_layout_conditions($variables) {
 }
 
 /**
+ * Output region content wrapped by tag specified in layout UI.
+ */
+function theme_layout_region_inner($variables) {
+  $tag = $variables['tag'];
+  $classes = $variables['classes'];
+
+  if ($tag != '') {
+    return '<' . $tag . ' class="' . $classes .'">' . implode('', $variables['blocks']) . '</' . $tag . '>';
```

Comment:
There aren't any attributes here.  Should I create an array in the theme function?  Something like this...
```php
function theme_layout_region_inner($variables) {
  $tag = $variables['tag'];
  $attributes = array('class' => explode($variables['classes']));

  if ($tag != '') {
    return '<' . $tag . ' ' . backdrop_attributes($attributes) .'>' . implode('', $variables['blocks']) . '</' . $tag . '>';
  }
  return implode('', $variables['blocks']);
}
```